ibeam/src/gateway_client.py

```diff
@@ -54,10 +54,7 @@ def __init__(self,
         then the environment values are assumed to be file
         paths to read for the secret value."""
 
-        if self.secrets_source == SECRETS_SOURCE_ENV:
-            _LOGGER.info('Reading secrets from the environment')
-        elif self.secrets_source == SECRETS_SOURCE_FS:
-            _LOGGER.info('Reading secrets from the filesystem')
+        _LOGGER.info('Secrets source: {self.secrets_source}')
```

Comment:
Cool! Just a small typo, we're missing an `f` before the opening quote so that it would read: `info(f'Secrets source...)`

Reply:
Oops, fixed.